flask_resty/pagination.py

```diff
@@ -528,12 +531,27 @@ def get_filter(
             for i in range(len(column_cursors))
         )
 
-    def get_filter_clause(self, column_cursors):
-        previous_clauses = sa.and_(
-            column == value for column, _, value in column_cursors[:-1]
-        )
+    @staticmethod
+    def get_previous_clause(column_cursors):
+        # Invoking and_() without arguments is deprecated, and will be disallowed in a future release.
+        # For an empty and_() construct, use and_(True, *args).
+        previous_clauses = sa.and_()
+        for column, _, value in column_cursors:
+            if value is None:
+                previous_clauses = sa.and_(previous_clauses, column.is_(None))
+            else:
+                previous_clauses = sa.and_(previous_clauses, column == value)
+        return previous_clauses
```

Comment:
does this produce something different compared to:

```py
return sa.and_(column.isnot_distinct_from(value) for column, _, value in column_cursors)
```

might avoid the empty and usage?

Reply:
good catch, thanks